.circleci/config.yml

```diff
@@ -411,6 +417,7 @@ jobs:
           name: SPM Release Build
           command: swift build -c release --target RevenueCat
           no_output_timeout: 30m
+      - slack-notify
```

Comment:
I would generally prefer these to be part of fastlane, but I think in this case, since there's an orb that gets updates and automatically post failures (right?), I like it here